deploy/kubernetes/resources/chirper/chirp-impl-statefulset.json

```diff
@@ -59,12 +59,8 @@
                 "value": "$HOSTNAME.chirpservice.default.svc.cluster.local"
               },
               {
-                "name": "AKKA_SEED_NODE_PORT",
-                "value": "2551"
-              },
-              {
-                "name": "AKKA_SEED_NODE_HOST",
-                "value": "chirpservice-0.chirpservice.default.svc.cluster.local"
+                "name": "AKKA_SEED_NODES",
+                "value": "chirpservice-0.chirpservice.default.svc.cluster.local:2551,chirpservice-1.chirpservice.default.svc.cluster.local:2551,chirpservice-2.chirpservice.default.svc.cluster.local:2551"
               },
```

Comment:
This changes the default from 1 node to 3. This could make it more difficult for some trying out this sample app to get started?

Reply:
Hmmm, I've re-read the description and I wonder if this change is actually necessary to support both `scale = 1` and `scale = 3` deployments.

Reply:
Thanks for the review! I'd like to clarify a few things based on your comments:

We can do both 1 and 3 replicas without this change (actually any number `n` replicas) -- `-1` and `-2` upto `-n` will all contact the `-0` pod to join the cluster.

The main problem this PR solves is to allow the `-0` pod to be rescheduled and rejoin the existing cluster, for instance if a node crashes or if it's deleted due to a rolling restart.

It doesn't make things more difficult to get started -- when the `-0` pod starts up, if it can't contact `-1` or `-2` it simply establishes a new cluster, so the behavior is unchanged in that regard.